### PR TITLE
Editor: Fixed sidebar glitch via h scrollable

### PR DESCRIPTION
**Pull request by ycw**
This PR fixed sidebar glitch(#28358) by using horizontal scrollable tabs in `UITabbedPanel`, this approach was suggested by @mrdoob https://github.com/mrdoob/three.js/pull/28358#issuecomment-2126250952:

https://github.com/mrdoob/three.js/assets/1063018/8587c377-0a4f-4219-b568-b4922eb514d8

Preview: https://raw.githack.com/ycw/three.js/editor-scrollable-tabsbar/editor/index.html

### CODE REVIEW

File: editor/js/Sidebar.Properties.js [hunk 1]
```diff
@@ -49,14 +49,17 @@ function SidebarProperties( editor ) {
 		if ( container.selected === 'geometryTab' ) {
 
 			container.select( geometryTab.isHidden() ? 'objectTab' : 'geometryTab' );
+			geometryTab.dom.scrollIntoView( { inline: 'center', behavior: 'smooth' } );
```
Comment (mrdoob) [inline editor/js/Sidebar.Properties.js:52]: That's nice! I didn't know about this 🙏

File: editor/js/libs/ui.js [hunk 2]
```diff
@@ -1130,6 +1130,7 @@ class UITabbedPanel extends UIDiv {
 		if ( tab ) {
 
 			tab.addClass( 'selected' );
+			tab.dom.scrollIntoView( { inline: 'center', behavior: 'smooth' } );
```
Comment (mrdoob) [inline editor/js/libs/ui.js:1133]: That's nice! I didn't know about this 🙏

Comment (ycw) [reply editor/js/libs/ui.js:1133]: congrats 🎉

Comment (ycw) [reply editor/js/libs/ui.js:1133]: Note that there's an issue when using `scrollIntoView` in the editor, and its solved by using `scrollTo` instead, see #28485 🙏🏻 sorry for that.

**Pull request merged**